templates/helpers.js

```diff
@@ -240,8 +241,7 @@ module.exports = function(docMap, options, getCurrent, helpers, OtherHandlebars)
                 return false;
             }
             var name = packageObject.name,
```

Comment:
Oh, I thought you were asking what is `packageObject`. For `packageObject.name`, that should simply be the `name` property, taken straight from `package.json`. So, for CanJS, it should be [just "`can`"](https://github.com/canjs/canjs/blob/master/package.json#L2).